app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -63,38 +63,37 @@ class CatalogueController @Inject()(
   environment: api.Environment,
   verifySignInStatus: VerifySignInStatus,
   umpAuthenticated: UmpAuthenticated,
-  override val runModeConfiguration: Configuration,
-  val messagesApi: MessagesApi)
-    extends FrontendController
-    with UserManagementPortalLink
-    with I18nSupport {
+  val serviceConfig: ServicesConfig,
```

Comment:
service**s**Config perhaps to keep it consistent

Reply:
What about making the `UserManagementPortalLink` a class with `ServicesConfig` injected into it (currently we need to inject `ServicesConfig` just because of that trait). Then, the portal link instance can be injected into the `CatalogueController` as a collaborator. It doesn't look very right to me that the `UserManagementPortalLink`'s api enriches the controller's api and you cannot say the `CatalogueController` is a `UserManagementPortalLink`.